cardano-testnet/src/Testnet/Components/SPO.hs

```diff
@@ -401,3 +405,45 @@ registerSingleSpo identifier tap@(TmpAbsolutePath tempAbsPath') nodeConfigFile s
               poolColdVkeyFp
               currentRegistedPoolsJson
   return (poolId, poolColdSkeyFp, poolColdVkeyFp, vrfSkeyFp, vrfVkeyFp)
+
+
+-- | Generates Stake Pool Operator (SPO) voting files (without signing)
+-- using @cardano-cli@.
+--
+-- This function takes the following parameters:
+--
+-- * 'execConfig': Specifies the CLI execution configuration.
+-- * 'work': Base directory path where the voting files and directories will be
+--           stored.
+-- * 'prefix': Name for the subfolder that will be created under 'work' to store
```

Comment:
Can you move this description to parameters' haddocks?

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/pull/5816/commits/68c08f3d21dac9a6dbab1be6fb21023f80898a1b

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/pull/5816/commits/f4deaa6250cf03576cc7c8267cc73c7295492ab7